CHANGELOG.md

```diff
@@ -1,5 +1,160 @@
+- Make `UTF-8` the default encoding for `application/json` (#6109) (Thanks @markekraus!)
+- Enable `$env:PAGER` to work correctly if arguments are used (#6144)
+
+### Test
+
+- Convert Web Cmdlets test to `one-true-brace-style` formatting (#5716) (Thanks @markekraus!)
+- Add a test for `IValidateSetValuesGenerator` in a module (#5830) (Thanks @iSazonov!)
+- Fix function to test for docker OS due to change to use `linuxkit` for macOS (#5843)
+- Replace `HttpListener` link header tests with `WebListener` (#5806) (Thanks @markekraus!)
+- Replace `HttpListener` echo tests with `WebListener` (#5840) (Thanks @markekraus!)
+- Replace `HttpListener` redirect tests with `WebListener` (#5872) (Thanks @markekraus!)
+- Stop `HttpListener` from running in Web Cmdlets tests (#5921) (Thanks @markekraus!)
+- Fix `PSVersion` in `PSSessionConfiguration` tests (#5554) (Thanks @iSazonov!)
+- Update test framework and tests to support Pester version 4 (#6064)
+- Update tests in `Microsoft.PowerShell.Management` folder to use Pester 4 Syntax. (#6294)
+- Add negative tests for `Copy-Item` over remote sessions (#6231)
+- Markdown test: Use strict in JavaScript (#6328)
+- Add tests for `Get-Process` about the `-Module` and `-FileVersion` parameters (#6272)
+- Add test for the `OsLocalDateTime` property of `Get-ComputerInfo`. (#6253)
+- Refactor Web Cmdlets Tests to Pester 4 Syntax (#6257) (Thanks @markekraus!)
+- Update tests in `Modules/CimCmdlets` folder to use Pester 4 Syntax. (#6306)
+- Update tests in `Language` folder to use Pester 4 Syntax. (#6304)
+- Change `Get-FileHash` tests to use raw bytes (#6430)
+- Remove `runas.exe` from tests as we have tags to control this behavior (#6432)
+- Update tests in `Engine` folder to use Pester 4 Syntax. (#6298)
+- Refactor the `Get-Content` tests to use `-TestCases`. (#6082)
+- Use `RequireAdminOnWindows` tag in `Set-Date` tests (#6034) (Thanks @stknohg!)
+- Remove `-TimeOutSec` from non timeout related tests (#6055) (Thanks @markekraus!)
+- Add verbosity and more accurate timeout implementation for `Start-WebListener` (#6013) (Thanks @markekraus!)
+- Skip tests that use `ExecutionPolicy` cmdlets on Unix (#6021)
+- Change Web Cmdlet tests to use `127.0.0.1` instead of `Localhost` (#6069) (Thanks @markekraus!)
+- Fix `Start-PSPester` to include or exclude `RequireSudoOnUnix` tag smartly on Unix (#6241)
+- Fix the terse output on Windows for test runs without admin privilege (#6252)
+- Add `RequireSudoOnUnix` tag for `Get-Help` tests. (#6223)
+- Add tests for `*-Item` Cmdlets in function provider (#6172)
+- Support running tests in root privilege on Linux. (#6145)
+
+### Build and Packaging Improvements
+
+- Add option to add shell explorer context menu in Windows installer (#5774) (Thanks @bergmeister!)
+- Make the shell explorer context menu registry entries platform specific to allow side by side of x86 and `x64`. (#5824) (Thanks @bergmeister!)
```

Comment:
Should be consistent with enclosing x86 in backticks.  Also should be `explorer shell`